euphony/src/main/java/co/euphony/tx/EuTxManager.java

```diff
@@ -140,13 +140,13 @@ public void euInitTransmit(String data) {
 	}
 
 	/*
-	 * @deprecated Replaced by {@link #setCode()}, deprecated for naming issue
+	 * @deprecated Replaced by {@link #play()}, deprecated for naming issue
 	 */
 	@Deprecated
 	public void process() { play(1, PlayerEngine.ANDROID_DEFAULT_ENGINE); }
 
 	/*
-	 * @deprecated Replaced by {@link #setCode()}, deprecated for naming issue
+	 * @deprecated Replaced by {@link #play()}, deprecated for naming issue
 	 */
 	@Deprecated
 	public void process(int count) { play(count, PlayerEngine.ANDROID_DEFAULT_ENGINE); }
```

Comment:
Actually, this comments are enough to express a reason why it was deprecated & substitution.
But, I think it would be good to separate it by clarifying the expression. (process(), process(int))

For process(int count),
```java
 @deprecated Replaced by {@link #play(int)}, deprecated for naming issue
```
How about this?
Similarly, For `setCode()` function, it will be better to express `setCode(String)`

Could you update both of them?